### ISSUE TITLE
Static framework fails to build with Xcode 10b3

### ISSUE THREAD

**Issue by friedbunny** [mapbox/mapbox-gl-native#12339]
Though building `iosapp` (and its constituent targets) succeeds in Xcode 10b3 at 19605cb4b4e73d3fe8bd1d3ec0a1e0bd8feb7f84, `make ipackage` fails when building the static framework, complaining about duplicate interfaces:

```
▸ Compiling MGLUserLocationHeadingBeamLayer.m

❌  /mapbox-gl-native/platform/ios/src/MGLUserLocationAnnotationView.h:14:1: duplicate interface definition for class 'MGLUserLocationAnnotationView'

@interface MGLUserLocationAnnotationView : MGLAnnotationView
^

❌  /mapbox-gl-native/platform/ios/src/MGLUserLocationAnnotationView.h:21:61: property has a previous declaration

@property (nonatomic, readonly, weak, nullable) MGLMapView *mapView;
```

/cc @jfirebaugh @1ec5

**Comment by 1ec5** [comment]
Also fails when building the dynamic+static scheme in Xcode. Looks like the CI scheme doesn’t build the static target as the dynamic+static scheme does.

**Comment by zugaldia** [comment]
Now that XCode 10 final is out, do we know if this is still an issue?

**Comment by 1ec5** [comment]
Yes, but there’s a fix in #12971.

**Comment by friedbunny** [comment]
After #12968, the static framework is now built using `make iframework FORMAT=static` — this compiles with Xcode 10, even without #12971 (as the issue appears to be with the `dynamic+static` scheme).
